docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,207 @@
+<!--
+Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the GPL, Version 3.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.gnu.org/licenses/gpl-3.0.txt
+-->
+
+# How to Contribute
+
+Become one of the contributors to this project! We strive to build a welcoming and open community for anyone who wants to use the project or contribute to it. There are just a few small guidelines you need to follow. To help us create a safe and positive community experience for all, we require all participants to adhere to the [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Table of Contents
+
+* [Become a contributor](#Become-a-contributor)
+* [Contributor agreement](#Contributor-Agreement)
+* [Submitting issues](#Submitting-issues)
+* [Triage issues](#Triage-issues)
+* [Your first contribution](#Your-first-contribution)
+* [Branching](#Branching)
+* [Signing your commits](#Signing-your-commits)
+* [Pull requests](#Pull-requests)
+* [Code reviews](#Code-reviews)
+* [Code Style](#Code Style)
```

Comment:
This link doesn't work. Please add a hypen between "Code" and "Style" to fix it.

Reply:
addressed as suggested